evm/src/test/scala/pravda/evm/translate/opcode/EvaluateTests.scala

```diff
@@ -0,0 +1,234 @@
+package pravda.evm.translate.opcode
+
+import fastparse.byte.all._
+import pravda.evm._
+import pravda.evm.EVM._
+import pravda.evm.translate.Translator
+import pravda.vm.Data.Primitive.BigInt
+import pravda.vm.VmSandbox.EnvironmentEffect.{StorageGet, StoragePut}
+import pravda.vm.VmSandbox.Preconditions
+import utest._
+
+import scala.collection.mutable.ArrayBuffer
+
+object EvaluateTests extends TestSuite {
+
+  val tests = Tests {
```

Comment:
See other tests in our project, you may want to write them as plaintext files. The new changes are coming, so I recommend you to wait some time.

Reply:
Now tests are merged, take a look at them. The new format will definitely improve readability of your tests.